### PR TITLE
176 street space

### PR DESCRIPTION
**Pull request by mvl22**
Adds the street space layer; refs #176.

### CODE REVIEW

File: images/clos_facilities.png
Comment (Robinlovelace) [inline images/clos_facilities.png:1]: :+1:

File: index.html [hunk 1]
```diff
@@ -146,7 +146,7 @@ <h2>Layer controls</h2>
 			<button class="accordion" id="autoopen">Route network</button>
 			<div class="panel">
 				<p>
-					<label><input type="checkbox" id="rnetcheckboxproxy" class="rnetproxy" checked>Show layer</label>
+					<label><input type="checkbox" id="rnetcheckboxproxy" class="rnetproxy" checked> Show layer</label>
```
Comment (Robinlovelace) [inline index.html:149]: Why the additional whitespace @mvl22 ?

File: index.html [hunk 2]
```diff
@@ -256,9 +256,23 @@ <h2>Layer controls</h2>
 				<div id="clos-legend" class="legend"></div>
 			</div>
 			
+			<button class="accordion">Street space</button>
+			<div class="panel">
+				<p>Analyis of the space available within each street.</p>
+				<p><label><input type="checkbox" class="showlayer" data-layer="streetspace" value="false"> Show LoS</label></p>
+				<div class="legend">
+					<div class="l_r">
+						<div class="lb"><span style="background-color: #dd7777;"></span>Wider than minimum</div>
```
Comment (Robinlovelace) [inline index.html:265]: :+1:

**Review by Robinlovelace — APPROVED** [review]
Great job, let's get it merged. @mvl22 please can you respond to the question or 2 post merge?

File: index.html (hunk 1 above)
Comment (mvl22) [reply index.html:149]: The label was too close to the box. Other such boxes didn't have this. Was just a quick tidy.

Comment (Robinlovelace) [reply index.html:149]: :+1:

**Pull request merged**